src/test/java/htsjdk/tribble/index/IndexFactoryTest.java

```diff
@@ -112,4 +117,45 @@ public void testCreateTabixIndexOnBlockCompressed() {
                     "Tabix indexed (bgzipped) VCF does not contain sequence: " + samSequenceRecord.getSequenceName());
         }
     }
+
+    @Test
+    public void testWriteIndex() throws Exception {
+        // temp directory for this test
+        final File tempDir = IOUtil.createTempDir("writeIndex", null);
```

Comment:
I think the standard in htsjdk is to use `File output = File.createTempFile()` to get your output file, and then call `output.deleteOnExit()` to make sure it's cleaned up afterwards.